entrypoint.sh

```diff
@@ -155,6 +156,8 @@ push_to_repository() {
     git branch -M master
     git remote add origin "https://oauth2:$github_token@github.com/$org_name/$repository_name.git"
     git push -u origin master
+    # add_link would be usable if not overwriting the existing links
+    # add_link "$repo_url"
```

Comment:
I don't understand what this comment is explaining.

Reply:
`add_link "$repo_url"` is pushing the repo URL to Port, which is nice. However, sadly this is overwriting the current URLs stored in Port. So we don't want to use it as it is working now.
I can make it an option instead.